.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow will do a clean install of node dependencies, build the source code and
+# run tests across different versions of node
+
+name: Build and Run Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  release:
+    types: [ published ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [ 8.x, 10.x, 11.x ]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+          registry-url: https://registry.npmjs.org/
+      - run: npm install
+      - run: npm run build --if-present
```

Comment:
Just wondering, what is the `--if-present` needed for here? Shouldn't it always have a `build` task?

Reply:
This is something the boiler plate generates automatically, but yes you are right the build task is present by default making this extraneous 😄